sites/fast-website/src/app/data/framework.data.ts

```diff
@@ -0,0 +1,58 @@
+export interface FrameworkContentPlacementData {
+    divider?: boolean;
```

Comment:
I would recommend alphabetizing inputs here

Reply:
will do.

Reply:
@awentzel I'm fine with the change being made here as @SamanthaAO is already on it, but some of these haven't been ironed out as hard/fast rules. Would you mind opening an issue to propose some of these code quality things so we can land what things are quality issues vs. preferences?